modules/scripts/dtbook-cleaner/src/main/resources/xml/dtbook-cleaner.script.xpl

```diff
@@ -0,0 +1,164 @@
+    </p:option>
+
+
+    <!-- Options from dtbook-fix script -->
+    <p:option name="repair" required="false" select="false()" px:type="boolean">
+        <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+            <h2 px:role="name">Repair the dtbook</h2>
+            <p px:role="desc">Apply repair routines on the dtbook</p>
+        </p:documentation>
+    </p:option>
+    <p:option name="fixCharset" required="false" select="false()" px:type="boolean">
+        <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+            <h2 px:role="name">Repair - Fix Charset</h2>
+            <p px:role="desc">Fix the document charset (To be implemented)</p>
+        </p:documentation>
+    </p:option>
+    <p:option name="tidy" required="false" select="false()" px:type="boolean">
+        <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+            <h2 px:role="name">Tidy up the dtbook</h2>
+            <p px:role="desc">Apply tidying routines on the dtbook</p>
+        </p:documentation>
+    </p:option>
+    <p:option name="simplifyHeadingLayout" required="false" select="false()" px:type="boolean">
+        <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+            <h2 px:role="name">Tidy - Simplify headings layout</h2>
+            <p px:role="desc">TBD</p>
+        </p:documentation>
+    </p:option>
+    <p:option name="externalizeWhitespace" required="false" select="false()" px:type="boolean">
+        <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+            <h2 px:role="name">Tidy - Externalize whitespaces</h2>
+            <p px:role="desc">TBD</p>
+        </p:documentation>
+    </p:option>
+    <p:option name="documentLanguage" required="false" select="''">
+        <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+            <h2 px:role="name">Tidy - Document language</h2>
+            <p px:role="desc">Set a document language</p>
+        </p:documentation>
+    </p:option>
+
+    <p:option name="narrator" required="false" select="false()" px:type="boolean">
+        <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+            <h2 px:role="name">Prepare dtbook for pipeline 1 narrator</h2>
+            <p px:role="desc">Apply pipeline 1 "narrator" cleaning routines on the document</p>
+        </p:documentation>
+    </p:option>
+    <p:option name="publisher" required="false" select="''" >
+        <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+            <h2 px:role="name">Narrator - Publisher</h2>
+            <p px:role="desc">Publisher to be added as dc:Publisher if none is defined in the dtbook</p>
+        </p:documentation>
+    </p:option>
+
+    <p:option name="ApplySentenceDetection" select="false()" px:type="boolean">
+        <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+            <h2 px:role="name">Apply sentences detection</h2>
+            <p px:role="desc">Encapsulate sentences within the document</p>
+        </p:documentation>
+    </p:option>
```

Comment:
@NPavie This option does not seem to be used?

Reply:
Sorry i had not seen your messages : 
this was to be used for sentence detection in post-processing of dtbook for the addin (readded in last commit)